peps/pep-0750.rst

```diff
@@ -182,6 +185,10 @@ Like ``Template``, it is a new concrete type found in the ``types`` module:
         ):
             ...
 
+Like ``Template``, ``Interpolation`` is immutable. Its attributes cannot be
+reassigned. (The ``value`` attribute may of course itself be mutable, but 
```

Comment:
`value` is immtuable, but it may point to a mutable object. Generally this is referred to as being shallow immutable.

Reply:
Yes, this is what was intended. I'll rewrite for clarity.